openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx

```diff
@@ -161,65 +164,73 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
     }, []);
 
     return (
-      <div className="tw-w-80">
-        <div className="tw-flex">
+      <div className="w-500">
+        <Space align="center" size="small">
           <div data-testid="owner">
```

Comment:
Can we give testid to the space component and remove this dive?

Reply:
@Sachin-chaurasiya `div` is the child of the Space element which represents the owner. so we can't assign the testid to Space and even `div` is the parent of 2 condition child. so we can't remove that too.